src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -87,6 +87,10 @@ public Tracing getTracing(final HttpTracing httpTracing) {
 
   @Provides @Singleton
   public HttpTracing getHttpTracing(final Config config, final ServerConfig serverConfig) {
```

Comment:
wonder if we can't use module override instead? https://stackoverflow.com/questions/483087/overriding-binding-in-guice